tests/e2e/performance/test_pvc_deletion_performance.py

```diff
@@ -127,13 +127,19 @@ def test_multiple_pvc_deletion_measurement_performance(self, teardown_factory):
         log.info("Preparing to delete 120 PVC")
 
         # Delete PVC
-        for obj in pvc_objs:
-            obj.delete()
+        helpers.delete_multiple_pvcs(pvc_objs)
         for obj in pvc_objs:
             obj.ocp.wait_for_delete(obj.name)
 
         # Get PVC deletion time
-        pvc_deletion_time = helpers.measure_pv_deletion_time_bulk(
-            interface=self.interface, pv_name_list=pv_name_list
-        )
-        logging.info(f"{number_of_pvcs} PVCs deletion time took {pvc_deletion_time}")
+        start_time = helpers.measure_multiple_pvc_deletion_time(self.interface, pvc_objs, status="start")
+        end_time = helpers.measure_multiple_pvc_deletion_time(self.interface, pvc_objs, status="end")
+        total = end_time - start_time
+        total_time = total.total_seconds()
```

Comment:
@SvetlanaRedHat 
having this in a separate function oud be great. 
get_pvc_deletion_status_time -> collects the start/end time
measure_pvc_deletion_time -> Does the diff between start time and end time. 

We don't need to expose get_pvc_deletion_status_time function and have it defined within measure_pvc_deletion_time. wdyt?

Reply:
Kartick I tried to make is similar to measure mutiple pvc creation function